index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li id="logo"><a href="#" class="logo">My Logo</a></li>
+        <li><img src="./img/sandwich.svg" alt="Menu Button" /></li>
+      </ul>
+    </header>
+    <div class="background-shapes">
+      <img src="./img/Ellipse-3.svg" class="ellipse-3" alt="" /><img
+        src="./img/Ellipse-4.svg"
+        class="ellipse-4"
+        alt=""
+      /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+        src="./img/Ellipse-8.svg"
+        class="ellipse-8"
+        alt=""
+      /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+        src="./img/Ellipse-12.svg"
+        class="ellipse-12"
+        alt=""
+      /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+        src="./img/Ellipse-14.svg"
+        class="ellipse-14"
+        alt=""
+      /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+        src="./img/Rectangle-4.svg"
+        class="rect-4"
+        alt=""
+      /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+        src="./img/Rectangle-6.svg"
+        class="rect-6"
+        alt=""
+      /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+    </div>
```

Comment:
- [ ] nice adding the background image, but it will be nice to follow [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) and add background image in our CSS file with `background-image: url('')`, therefore kindly remove this `div` as it will be implemented from CSS file

Reply:
Actually, it's impossible to have an empty div with background images, I've tried it. 
And the project requirement actually specified both the "images and backgrounds", so we are free to use them both.
That was a good one! Thanks